Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CovidCertificateConfigMapper.kt

```diff
@@ -12,16 +13,19 @@ import javax.inject.Inject
 
 @Reusable
 class CovidCertificateConfigMapper @Inject constructor() : CovidCertificateConfig.Mapper {
-    override fun map(rawConfig: AppConfigAndroid.ApplicationConfigurationAndroid): CovidCertificateConfig {
-        if (!rawConfig.hasDgcParameters()) {
-            Timber.w("Config has no DCC parameters.")
-            return CovidCertificateConfigContainer()
-        }
+    override fun map(rawConfig: AppConfigAndroid.ApplicationConfigurationAndroid): CovidCertificateConfig = try {
+        if (!rawConfig.hasDgcParameters()) throw IllegalStateException("Config has no DCC parameters.")
```

Comment:
would that affect other features using `CovidCertificateConfigContainer` too?

Reply:
Yes, that's why I requested a new default App config to ensure the app keeps working as expected afterwards

Reply:
Un-resolve , until the default config is in place